internal/pkg/cli/task_run.go

```
@@ -681,6 +681,10 @@ Did you tag your secrets with the "copilot-application" and "copilot-environment
 		if err := o.displayLogStream(); err != nil {
 			return err
 		}
+		if exitCode, err := o.runner.CheckNonZeroExitCode(tasks); err != nil {
+			fmt.Println(err)
```

Do we intend to print the error out to users? If so what do you think of `log.Error`?

---

Yes we are planning on displaying error to the customer instead of just silently failing the task. `log.error` makes sense.